<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterChainPanel</class>
 <widget class="QWidget" name="FilterChainPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QPushButton" name="addFilterButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Добавить фильтр</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="filterListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>288</width>
        <height>553</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#filterListWidget {
    background-color: #F8F9FA;
}</string>
      </property>
      <layout class="QVBoxLayout" name="filterListLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
